### PR TITLE
voting now initiated at round start

### PR DESCRIPTION
**Pull request by MattSScott**
Loot voting seems to be fixed, based on tests

### CODE REVIEW

File: pkg/infra/teams/team3/loot_strat.go [hunk 1]
```diff
@@ -60,10 +61,12 @@ func (a *AgentThree) LootAction(
...
+// this is really poor from infra - i'd just ignore tbh
 func (a *AgentThree) HandleLootInformation(m message.TaggedInformMessage[message.LootInform], baseAgent agent.BaseAgent) {
 	// submit a proposal to the leader
+	fmt.Println("Made it here")
 	switch m.Message().(type) {
-	case *message.StartLoot:
+	case message.LootInform:
 		// Send Proposal?
 		sendProposal := rand.Intn(100)
 		if sendProposal < a.personality {
```
Comment (MattSScott) [inline pkg/infra/teams/team3/loot_strat.go:72]: This function was never being called, which in turn meant that no proposals were ever being sent. I've added a driver function (below) in `RequestLootProposal`

File: pkg/infra/teams/team3/loot_strat.go [hunk 2]
```diff
@@ -75,9 +78,18 @@ func (a *AgentThree) HandleLootInformation(m message.TaggedInformMessage[message
...
+// forcibly call at start of loot phase to begin proceedings
+func (a *AgentThree) RequestLootProposal(baseAgent agent.BaseAgent) { // put your logic here now, instead
+	sendProposal := rand.Intn(100)
+	if sendProposal > a.personality {
+		return
+	}
+	// general and send a loot proposal at the start of every turn
+	baseAgent.SendLootProposalToLeader(a.generateLootProposal())
+}
+
```
Comment (MattSScott) [inline pkg/infra/teams/team3/loot_strat.go:90]: This function now needs to be implemented. It is called at the start of each round and asks for the agent to send a loot proposal. The leader then takes this and distributes it for voting, which subsequently calls the `HandleLootProposal`

File: pkg/infra/teams/team3/loot_strat.go [hunk 3]
```diff
@@ -75,9 +78,18 @@ func (a *AgentThree) HandleLootInformation(m message.TaggedInformMessage[message
...
 func (a *AgentThree) HandleLootProposal(_ message.Proposal[decision.LootAction], _ agent.BaseAgent) decision.Intent {
```
Comment (MattSScott) [inline pkg/infra/teams/team3/loot_strat.go:91]: Put your voting logic inside of here

**Review by Noanan — COMMENTED** [review]
LGTM

**Pull request merged**